source/reference/limitations.txt

```diff
@@ -132,10 +132,19 @@ Reversing
 Multiple Clusters
 -----------------
 
-- Syncing multiple source clusters to one destination cluster isn't
-  supported.
-- Syncing one source cluster to many destination clusters isn't
-  supported.
+- ``mongosync`` does not support syncing multiple source clusters to one
+  destination cluster. 
+- One ``mongosync`` instance cannot sync a source cluster to many
```

Comment:
I don't think that this point is needed. Its a pretty obvious limitation that I don't think needs to be called out. I think it does make sense to call out the thing in the note below somewhere.

Reply:
Because this page previously said that mongosync **cannot** support 1 to N replication so I think it might be important to note this, at least for a while, in case users become confused with the change

Reply:
To me, it doesn't seem that users would be attached to that previous limitation, or remember it. I think its more confusing to add that point, and I'd rather the docs just be as clear as possible in their current form.

Reply:
That makes sense, I removed the bullet. Could you give this another review when you get a chance?